wiki/Tournaments/otst!/2019/en.md

```diff
@@ -106,94 +108,55 @@ The osu! TV Size Tournament 2019 is run by various osu! community members.
 
 -----------------------
 
-## Match Schedules: Group Stage
-
-### Saturday, 2 March 2019
-
-| Match ID | Team A | Team B | Match Time (UTC) |
-| :-: | --: | :-- | :-: |
-| #A3 | =yahe | all waifu hero | **13:00** |
-| #B6 | budget aussies | ok\_hand osu | **13:00** |
-| #B4 | BRATVIA I KOLCO | budget aussies | **14:00** |
-| #B1 | Rubber Conquest | BRATVIA I KOLCO | **19:00** |
-| #B5 | BRATVIA I KOLCO | ok\_hand osu | **19:00** |
-| #B3 | Rubber Conquest | ok\_hand osu | **20:00** |
-
-### Sunday, 3 March 2019
+## Match Schedules: Round of 16
 
-| Match ID | Team A | Team B | Match Time (UTC) |
-| :-: | --: | :-- | :-: |
-| #A4 | Roselia | our angel kodama | **01:00** |
-| #D6 | Gay Squad | tibi saised | **01:00** |
-| #A1 | =yahe | Roselia | **02:00** |
-| #A2 | =yahe | our angel kodama | **03:00** |
-| #A5 | Roselia | all waifu hero | **03:00** |
-| #A6 | our angel kodama | all waifu hero | **04:00** |
-| #C4 | Fine Lady Squad | Raise My PeePee | **04:00** |
-| #B2 | Rubber Conquest | budget aussies | **05:00** |
-| #D3 | #arsecrackers | tibi saised | **05:00** |
-| #C1 | Riven small PP | Fine Lady Squad | **15:00** |
-| #C5 | Fine Lady Squad | RNG Game | **16:00** |
-| #D1 | #arsecrackers | la profanacion | **16:00** |
-| #D2 | #arsecrackers | Gay Squad | **17:00** |
-
-### Monday, 4 March 2019
+### Saturday, 16 March 2019
 
 | Match ID | Team A | Team B | Match Time (UTC) |
 | :-: | --: | :-- | :-: |
-| #C2 | Riven small PP | Raise My PeePee | **01:00** |
-| #D4 | la profanacion | Gay Squad | **01:00** |
-| #D5 | la profanacion | tibi saised | **02:00** |
-| #C6 | Raise My PeePee | RNG Game | **03:00** |
+| #W11 | =yahe | canvas garden | **12:00** |
+| #W12 | Rubber Conquest | bruh | **14:00** |
+| #W13 | Riven small PP | HDcarryEmertxE | **17:00** |
+| #W16 | Hotel for Lolis | Raise My PeePee | **22:00** |
 
-### Tuesday, 5 March 2019
+### Sunday, 17 March 2019
 
 | Match ID | Team A | Team B | Match Time (UTC) |
 | :-: | --: | :-- | :-: |
-| #C3 | Riven small PP | RNG Game | **00:00** |
+| #W14 | #arsecrackers | Valhalla | **12:00** |
+| #W18 | Floating up | Roselia | **16:00** |
+| #W17 | osu | BRATVA I KOLCO | **19:00** |
+| #W15 | derust | la profanacion | **20:00** |
 
-### Saturday, 9 March 2019
+-----------------------
```

Comment:
these separators probably aren't necessary since there is a higher-level header right below it (applies to a lot of articles I'm sure, just something I noticed)